pyiron/sphinx/interactive.py

```diff
@@ -45,6 +46,10 @@ def structure(self):
     @structure.setter
     def structure(self, structure):
         GenericInteractive.structure.fset(self, structure)
+        if structure is not None:
```

Comment:
Do we have to do this twice (in base and interactive)?

Reply:
The issue is that SphinxInteractive is derived from SphinxBase and GenericInteractive so we could remove it from SphinxBase as in principle nobody should use just SphinxBase but for now I did not want to break it and preferred to implement it twice.